package.json

```diff
@@ -163,11 +163,11 @@
     "react-native-tab-view": "^3.5.2",
     "react-native-url-polyfill": "^2.0.0",
     "react-native-view-shot": "3.8.0",
-    "react-native-vision-camera": "2.16.8",
+    "react-native-vision-camera": "github:j-piasecki/react-native-vision-camera#051c9c03a347c1cbf16d8e4ff1a9c4a656567ab3",
```

Comment:
same here, we need to use patches instead. That said, we work closely with @mrousavy and can ask him to review a PR?

Reply:
I doubt it will get merged, I adapted changes from https://github.com/mrousavy/react-native-vision-camera/pull/1684 to work on newer versions of RN, but this PR removes frame processors support (which are not used by the App).

Reply:
we can use patches in the meantime as long as we've got issues open to follow-up with upstream fixes